quiz.html

```diff
@@ -3,53 +3,121 @@
+							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question2" value="Dry">Dry</p></div>
+							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question2" value="Normal">Normal</p></div>
+							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question2" value="Sensitive">Sensitive</p></div>
+							<hr color="white" width="80%" size="3">
+						</div>
+					</div>
+		
+					<div>
+						<h1>Lorem ipsum</h1>
+						<h2>3 / 5</h2>
+						<div class="choices">
+							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question3" value="Oily">Oily</p></div>
+							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question3" value="Dry">Dry</p></div>
+							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question3" value="Normal">Normal</p></div>
+							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question3" value="Sensitive">Sensitive</p></div>
+							<hr color="white" width="80%" size="3">
+						</div>
+					</div>
+		
+					<div>
+						<h1>Lorem ipsum</h1>
+						<h2>4 / 5</h2>
+						<div class="choices">
+							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question4" value="Oily">Oily</p></div>
+							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question4" value="Dry">Dry</p></div>
+							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question4" value="Normal">Normal</p></div>
+							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question4" value="Sensitive">Sensitive</p></div>
+							<hr color="white" width="80%" size="3">
+						</div>
+					</div>
+		
+					<div>
+						<h1>Lorem ipsum</h1>
+						<h2>5 / 5</h2>
+						<div class="choices">
+							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question5" value="Oily">Oily</p></div>
+							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question5" value="Dry">Dry</p></div>
+							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question5" value="Normal">Normal</p></div>
+							<div class="choicebox"><div class="placeholder"></div><p><input type="radio" name="question5" value="Sensitive">Sensitive</p></div>
+							<hr color="white" width="80%" size="3">
+						</div>
+					</div>
+		
+					<input type="button" name="" id="submit" value="Submit" onclick="check()">
+		
+				</form>
 
-					<h1>Question 5 heading</h1>
-					<h2>5 / 5</h2>
-					<p>Question related images and text will go here</p>
-					
-					<button>Submit</button>
+				<!-- This div is hidden until user clicks submit button -->

```

Comment:
This screen only shows up once the quiz has been completed, and the outcome differs based on the user's answers. The page is still the same (quiz.html), but certain content hides while other content shows up in order to achieve this outcome.